ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/IdentityProviderConfigValidation.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.apiserver.services.admin
+
+import com.daml.ledger.api.domain.IdentityProviderId
+import com.daml.logging.LoggingContext
+import com.daml.platform.localstore.api.IdentityProviderConfigStore
+
+import scala.concurrent.Future
+
+trait IdentityProviderConfigValidation {
+
+  def identityProviderConfigExists(id: IdentityProviderId)(implicit
+      loggingContext: LoggingContext
+  ): Future[Boolean]
+
+}
+
+class IdentityProviderConfigValidationImpl(
+    identityProviderConfigStore: IdentityProviderConfigStore
+) extends IdentityProviderConfigValidation {
+
+  override def identityProviderConfigExists(id: IdentityProviderId)(implicit
```

Comment:
```suggestion
  override def exists(id: IdentityProviderId)(implicit
```
? because it's a simple one method API

Reply:
I made it `apply` which simplifies it even further